UnityProject/Assets/Scripts/Tilemaps/Behaviours/Layers/MetaTileMap.cs

```diff
@@ -260,13 +263,10 @@ public void UpdateMe()
 
 		private void ApplyTileChange(TileLocation tileLocation)
 		{
+
 			lock (QueuedChanges)
 			{
-				if (QueuedChanges.Contains(tileLocation))
-				{
-					return;
-				}
-
+				if (tileLocation.InQueue) return;
```

Comment:
Need to set `tileLocation.InQueue` true after this If?

Reply:
Didn't catch this, thanks